covers/pipelines.py

```diff
@@ -44,14 +44,36 @@ def store_item(self, item):
         opponent = item['opponent']
         date = item['date']
         location = item['location']
-
         # covers.com has an error and lists a Houston @ Sacramento game as having taken place in Houston
         if opponent == 'Houston' and date == '04/04/95':
             location = 'vs'
-
+        #apparently this data is needed to log it to the database?
+        if '@' in location:
+            location='vs'
+        else:
+            location='vs'
+        season_type = item['season_type']
+        season_type = 'Regular Season'
```

Comment:
you're right I probably should have just hard coded it or found a better way. It doesn't look like the regular season is even a value given by the website anymore. I was trying to use as much of the existing code as possible without making too many modifications because I didn't have 100% full understanding of all the components inner workings. So many of your suggestions are valid and I will make the changes.